cmd/utils/flags.go

```diff
@@ -812,6 +812,12 @@ var (
 		Name:  "l1.sync.startblock",
 		Usage: "L1 block height to start syncing from. Should be set to the L1 message queue deployment block number.",
 	}
+
+	// Circuit capacity check settings
+	CircuitCapacityCheckEnabledFlag = cli.BoolFlag{
+		Name:  "checkcircuitcapacity",
+		Usage: "Enable circuitcapacity check in block validation",
```

Comment:
Maybe we could also just use `ccc` (similar to `gpo`).

```suggestion
		Name:  "ccc",
		Usage: "Enable circuit capacity check during block validation",
```